mcs/class/corlib/System.IO/FileSystem.NotSupported.cs

```diff
@@ -0,0 +1,32 @@
+namespace System.IO
+{
+	internal static partial class FileSystem
+	{
+		public static void CopyFile(string sourceFullPath, string destFullPath, bool overwrite) => throw new PlatformNotSupportedException ();
+		public static void ReplaceFile(string sourceFullPath, string destFullPath, string destBackupFullPath, bool ignoreMetadataErrors) => throw new PlatformNotSupportedException ();
+		public static void MoveFile(string sourceFullPath, string destFullPath) => throw new PlatformNotSupportedException ();
+		public static void DeleteFile(string fullPath) => throw new PlatformNotSupportedException ();
+		public static void CreateDirectory(string fullPath) => throw new PlatformNotSupportedException ();
+		public static void MoveDirectory(string sourceFullPath, string destFullPath) => throw new PlatformNotSupportedException ();
+		public static void RemoveDirectory(string fullPath, bool recursive) => throw new PlatformNotSupportedException ();
+		public static bool DirectoryExists(string fullPath) {
+			MonoIOError error;
```

Comment:
Why is this method special?

Reply:
It was used by gensources while on the basic profile, so the method needed to work. Change removed as basic profile is no more